### DISCUSSION TITLE
Does this android app allow adding and updating a git password store?

### DISCUSSION THREAD

**Discussion by ccaneke** [zeapo/Android-Password-Store#3201] [Q&A]
I'm looking for an android app to enable me to use pass on my mobile phone.
I'd like to confirm whether this app allows adding, updating, and reading passwords in a git password store? If so does it require the GPG Key that was used to lock the password store in a remote git repo?

**Comment by msfjarvis** [comment]
It does support all those operations and it does need the GPG key(s) used for encrypting the store.
